scripts/start_audio.sh

```diff
@@ -1,6 +1,6 @@
 export LD_LIBRARY_PATH=/usr/local/lib/
 cd $HOME/
-gst-launch-1.0 -v -e alsasrc device=hw:1,0 ! audioconvert ! rtpL16pay ! udpsink host=192.168.2.1 port=5601
+gst-launch-1.0 -v -e alsasrc device=hw:1,0 ! audioconvert ! rtpL16pay ! udpsink host=192.168.2.255 port=5601
```

Comment:
todo

Reply:
we should probably disable audio streaming for the moment, as we don't support it topside. I think we should not enable broadcast due to the bandwidth and issues we encountered before with the network breaking down while broadcasting too much data.